libraries/AP_Param/AP_Param.cpp

```diff
@@ -1947,6 +1947,17 @@ void AP_Param::convert_old_parameters(const struct ConversionInfo *conversion_ta
     flush();
 }
 
+// convert old vehicle parameters to new object parameters with scaling - assumes all parameters will have the same scaling factor
+void AP_Param::convert_old_parameters(const struct ConversionInfo *conversion_table, uint8_t table_size, float scaler, uint8_t flags)
```

Comment:
```suggestion
void AP_Param::convert_old_parameters_scale(const struct ConversionInfo *conversion_table, uint8_t table_size, float scaler, uint8_t flags)
```

We trying to avoid relying on the argument fingerprint as it can be confusing.

Reply:
the existing method should call straight into this one with a 1.0 as the scale.

Reply:
Not sure I agree, it's a pretty standard way to write C++, but if that's the standard happy to go with it.

Reply:
Done